client/src/leap/soledad/client/adbapi.py

```diff
@@ -217,6 +217,9 @@ def _runU1DBQuery(self, trans, meth, *args, **kw):
         """
         meth = getattr(trans, meth)
         return meth(*args, **kw)
+        # XXX should return a fetchall?
```

Comment:
hmm what do you mean by turning asyncthe u1db sqlite plugin? do you think the current solution is not enough?

my comment here goes in the direction of making a separation between run_query and run_operation. for some reason when I started playing with this I didn't pay attention to the distinction between both (one makes a fetchall in the original api, and the operation doesn't), and all the sqlcipher transactions in our modification are using the "query" instead of the "operation".